pkgs/by-name/pd/pdfium-binaries/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+}:
+let
+  version = "6721";
+  src =
+    let
+      inherit (stdenv.hostPlatform) system;
+      selectSystem = attrs: attrs.${system} or (throw "Unsupported system: ${system}");
+      suffix = selectSystem {
+        x86_64-linux = "linux-x64";
+        aarch64-linux = "linux-arm64";
+        x86_64-darwin = "mac-x64";
+        aarch64-darwin = "mac-arm64";
+      };
+      hash = selectSystem {
+        x86_64-linux = "sha256-h4Pg5G2EYSUie3c11WJcKESiX/MVwL9gooNNRXYiuSE=";
+        aarch64-linux = "sha256-oAmM+alF7047HqfHd3B+nFPYw8GRiJOnLPH8bD6RnBQ=";
+        x86_64-darwin = "sha256-SC+sG54j3qb5ON4bBS1IWB0hLFXUihA2k6SZsZjO+Dc=";
+        aarch64-darwin = "sha256-uiEMupBSl6QPWHbzs8vyC/JHl5cvHuwoGtC+2cGqR64=";
+      };
+    in
+    fetchurl {
```

Comment:
Why not use `fetchzip` if you have to unpack it anyway? (Contrary to its name, `fetchzip` supports many more formats than just zip)